app/models/relationship.rb

```diff
@@ -7,4 +7,14 @@ class Relationship < ApplicationRecord
   validates :follower_id, presence: true
   validates :followed_id, presence: true
   validates :status, presence: true
+
+  after_create :send_relationship_notification
```

Comment:
`after_create :send_relationship_notification, unless: -> { Rails.env.test? }`
it takes you ability to test `send_relationship_notification` method